examples/CustomAssignmentConsoleApp/appsettings.json

```diff
@@ -0,0 +1,31 @@
+﻿{
+  "FeatureManagement": {
+    "DailyDiscount": {
+      "Assigner": "DayOfWeek",
+      "Variants": [
+        {
+          "Default": true,
+          "Name": "Default",
+          "ConfigurationReference": "DailyDiscount:Default"
+        },
+        {
+          "Name": "Default",
+          "ConfigurationReference": "DailyDiscount:Tuesday",
+          "AssignmentParameters": {
+            "DayOfWeek": "Tuesday"
```

Comment:
>DayOfWeek [](http://example.com/codeflow?start=13&length=9)

Can we change the name of this assignment parameter? Since "DayOfWeek" is also the name of current "Assigner", its a bit confusing.

Reply:
I see. `Value`?

Reply:
If you need a more specific name, how about `WeekDay`?

Reply:
I believe the initial confusion was caused by the repeated name. After some thought I reworked it a bit and I think the confusion is gone and it's even clearer.

```

    "DailyDiscount": {
      "Assigner": "Recurring",
      "Variants": [
        {
          "Default": true,
          "Name": "Default",
          "ConfigurationReference": "DailyDiscount:Default"
        },
        {
          "Name": "Default",
          "ConfigurationReference": "DailyDiscount:Tuesday",
          "AssignmentParameters": {
            "Days": [
              "Tuesday"
            ]
          }
        }
      ]
    }
  },
```